lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -373,6 +373,20 @@ testRule({
 			description: 'explicit scroll-padding test',
 			message: messages.expected('scroll-padding'),
 		},
+		{
+			code: 'a { grid-template-columns: repeat(3, 1fr); grid-template-rows: auto; grid-template-areas: "left center right"; }',
```

Comment:
nit

Only one declaration is necessary.
ditto for l384
e.g. `a { grid-template-columns: repeat(3, 1fr); }`

Reply:
Hm, I don't think that will work: it won't trip the rule (it requires *all* of the items represented by the shorthand to fire), and so no error will be reported by the rule. I just tried this locally to confirm. Did you mean something else?